src/modules/item/sagas.ts

```diff
@@ -90,7 +116,26 @@ import { getData as getItemsById, getItems, getEntityByItemId, getCollectionItem
 import { ItemTooBigError } from './errors'
 import { buildZipContents, getMetadata, isValidText, MAX_FILE_SIZE, toThirdPartyContractItems } from './utils'
 
-export function* itemSaga(builder: BuilderAPI) {
+const takeLatestCancellable = <A extends Action>(
```

Comment:
I feel like this should maybe live in `common/sagas.ts` ? (looks really cool btw)